src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -125,6 +127,20 @@ boot_newcomp_create(spdid_t spdid, struct cos_compinfo *comp_info)
 
 	boot_newcomp_sinv_alloc(spdid);
 
+	vmutpt = cos_pgtbl_alloc(&boot_info);
```

Comment:
130-140 looks like a limited range of the capabilities that you're adding in here.  Any reason you chose these, and not others?  (I assume because this was "sufficient to get it working", but I'm curious if there was more thought put into it.)

Reply:
This was purely based on what was sufficient, literally added caps till it booted... 

would be interested to discuss (forgot to bring this up on friday) what the full capability range actually is